src/components/Property/Contacts/TenantsContactTable/TenantPhoneNumber.js

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react'
+import RemoveContactModal from './modals/RemoveContactModal'
+import SetAsMainModal from './modals/SetAsMainModal'
+import classNames from 'classnames'
+
+const TenantPhoneNumber = ({ phoneNumber, tenant, reloadContacts }) => {
```

Comment:
As we're now effectively coupling MMH and RepairsHub, how about an ErrorBoundary around these likely-to-break components?

Reply:
What part of this code do you see as likely to break? To summarise the changes:

- Its using the existing integration with the contact details endpoint to display them
- A link to MMH, which is populated by an environment variable
- PATCH and DELETE requests called from within confirmation modals. These contain error handling

I'm not sure what part you think needs an error boundary.